api/services/authService.js

```diff
@@ -0,0 +1,94 @@
+module.exports = {
+  authService: function() {   
```

Comment:
Rather than exposing "authService" I would expose the two functions (redirectToGitHub and authenticate) that you want people to use.  Will save the need to then call "authService()" after the module is required.